packages/charts/src/vaadin-chart.js

```diff
@@ -579,7 +577,6 @@ class Chart extends ResizeMixin(ElementMixin(ThemableMixin(PolymerElement))) {
     }
 
     if (this.categories) {
-      options.xAxis ||= {};
```

Comment:
Removed this line as not needed here and below, as it's present in the base config:

https://github.com/vaadin/web-components/blob/ed28a0f1f3b2a40ed26c25ffbfe915a1487b9cc3/packages/charts/src/vaadin-chart.js#L503